index.html

```diff
@@ -599,9 +604,9 @@
 [task source]: http://www.w3.org/TR/2011/WD-html5-20110525/webappapis.html#task-source
 [event loop]: http://www.w3.org/TR/html5/webappapis.html#event-loop
 [DOMHighResTimestamp]: http://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp
-[performance.now()]: http://www.w3.org/TR/hr-time-2/#widl-Performance-now-DOMHighResTimeStamp
 [pv-hidden]: http://www.w3.org/TR/page-visibility/#dom-document-hidden
 [fully active]: http://www.w3.org/TR/html5/browsers.html#fully-active
 [list of active timers]: http://www.w3.org/TR/html5/webappapis.html#list-of-active-timers
 [requestAnimationFrame]: http://www.w3.org/TR/animation-timing/#dom-windowanimationtiming-requestanimationframe
 [report the error]: http://www.w3.org/TR/html5/webappapis.html#report-the-error
+[internal slot]: https://tc39.github.io/ecma262/#sec-object-internal-methods-and-internal-slots
```

Comment:
You probably don't want to reference this. Most other standards also don't say "internal slot" at the moment. They just say that the object has an associated X (where X is a name).